lib/order.rb

```diff
@@ -0,0 +1,69 @@
+require 'csv'
+require_relative 'customer'
+
+class Order
+
+  attr_reader :id
+  attr_accessor :products, :fulfillment_status, :customer
+ 
+  def initialize(id, products, customer, fulfillment_status = :pending)
+    @id = id.to_i
+    @products = products
+    @fulfillment_status = fulfillment_status
+    @customer = customer
+
+    valid_stats = [:pending, :processing, :paid, :shipped, :complete]
+    
+    unless valid_stats.include? fulfillment_status
+      raise ArgumentError.new("The given fulfillment status is invalid, please try again.")
+    end
+
+  end
+
+  def total
+    if @products == {}
+     return 0 
+    end
+
+    pre_tax_total = @products.values.sum
+    final_total = pre_tax_total * 1.075
+
+    return ('%.2f' % final_total).to_f
```

Comment:
If you'd just like to round a `Float` to two digits past the decimal point they have a `round` method.

```suggestion
    return final_total.round(2)
```